Indicator.struct.h

```diff
@@ -116,9 +116,12 @@ struct IndicatorCalculateCache {
 };
 
 #ifndef __MQLBUILD__
-// The structure of input parameters of indicators.
+/* The structure of input parameters of indicators.
```

Comment:
You started a comment, where is the end `*/`? This breaks the syntax of the entire file.